PULSEDEV-00000 feedzai-openml: Include release process in README (#67)
feedzai/feedzai-openml: README.md

This project makes use of the [jgitver Maven plugin](https://github.com/jgitver/jgitver). When using Intellij IDEA you
must configure the project to skip the plugin altogether. [See the related issue](https://github.com/jgitver/jgitver-maven-plugin/wiki/Intellij-IDEA-configuration).

## Releasing

> If the hotfix branch is ready, you needed to create an annotated tag pointing to the hotfix branch head (example below for releasing version 1.2.29):

Reviewer: Just a small rewording suggestion (the current wording is fine too, so no objections if you prefer it):

```suggestion
If the hotfix branch is ready to be published, you needed to create an annotated tag pointing to the head of the hotfix branch. Below is an example for releasing version 1.2.29:
```

```bash

> # Ensure the tag is made on the udpated branch

Reviewer: ```suggestion
# Ensure the branch is up-to-date before applying the tag
```

git fetch -a
git checkout origin/hf-1.2.X
git tag -a 1.2.29

> # Your EDITOR will open. Write a good message and save as it is used on Github as a release message

Reviewer: ```suggestion
# The command above will launch your default CLI text editor.
# Write an informative message and save the contents, which will be used on GitHub as a release message.
```
